src/app/submission/submission-edit/submission-edit.component.html

```diff
@@ -24,6 +24,9 @@
             be available on BioStudies but will not be reflected in the initially submitted file.
           </p>
         </div>
+        <div *ngIf="banner" class="card" [style.background]="banner.backgroundColor" style="justify-content: center; padding: 5px 0; margin: 0 0 10px 0">
```

Comment:
I would suggest using the classes provided by bootstrap. In this case the inline styles can be translated to this classes:
 
```html
<div class="card d-flex justify-content-center px-2 mb-3"... >
```